server/routes/serviceProviderReferrals/interventionProgressPresenter.test.ts

```diff
@@ -87,6 +87,60 @@ describe(InterventionProgressPresenter, () => {
       expect(presenter.sessionTableRows).toEqual([])
     })
 
+    describe('when there are multiple sessions', () => {
+      it('they must be ordered correctly', () => {
+        const referral = sentReferralFactory.build()
+        const actionPlan = actionPlanFactory.submitted().build({ id: '77923562-755c-48d9-a74c-0c8565aac9a2' })
+        const intervention = interventionFactory.build()
+        const presenter = new InterventionProgressPresenter(
+          referral,
+          intervention,
+          actionPlan,
+          [
+            actionPlanAppointmentFactory.build({ sessionNumber: 2 }),
+            actionPlanAppointmentFactory.build({ sessionNumber: 3 }),
+            actionPlanAppointmentFactory.build({ sessionNumber: 1 }),
+          ],
+          supplierAssessmentFactory.build()
+        )
+        expect(presenter.sessionTableRows).toEqual([
+          {
+            sessionNumber: 1,
```

Comment:
_suggestion_ what do you think about mapping to the sessionNumber, so the assertion does not fail on other, unrelated content changes in the future?

Reply:
excellent suggestion